malariagen_data/anoph/h12.py

```diff
@@ -514,6 +514,305 @@ def plot_h12_gwss(
+        ] = h12_params.max_cohort_size_default,
+        sample_query_options: Optional[base_params.sample_query_options] = None,
+        sample_sets: Optional[base_params.sample_sets] = None,
+        random_seed: base_params.random_seed = 42,
+        sizing_mode: gplt_params.sizing_mode = gplt_params.sizing_mode_default,
+        width: gplt_params.width = gplt_params.width_default,
+        track_height: gplt_params.track_height = 170,
+        genes_height: gplt_params.genes_height = gplt_params.genes_height_default,
+        show: gplt_params.show = True,
+        output_backend: gplt_params.output_backend = gplt_params.output_backend_default,
+    ) -> gplt_params.figure:
+        cohort_queries = self._setup_cohort_queries(
+            cohorts=cohorts,
+            sample_sets=sample_sets,
+            sample_query=sample_query,
+            sample_query_options=sample_query_options,
+            cohort_size=cohort_size,
+            min_cohort_size=None,
+        )
+
+        if isinstance(window_size, int):
+            window_size = {k: window_size for k in cohort_queries.keys()}
+        elif isinstance(window_size, dict):
+            if np.sort(window_size.keys()) != np.sort(cohort_queries.keys()):
+                raise ValueError("Cohorts and window_sizes should have the same keys.")
+
+        # Plot GWSS track.
+        figs: list[gplt_params.def_figure] = []
+        for i, (cohort_label, cohort_query) in enumerate(cohort_queries.items()):
+            params = dict(
+                contig=contig,
+                analysis=analysis,
+                window_size=window_size[cohort_label],
+                sample_sets=sample_sets,
+                sample_query=cohort_query,
+                cohort_size=cohort_size,
+                min_cohort_size=min_cohort_size,
+                max_cohort_size=max_cohort_size,
+                random_seed=random_seed,
+                title=cohort_label,  # Deal with a choice of titles later
+                sizing_mode=sizing_mode,
+                width=width,
+                height=track_height,
+                show=False,
+                output_backend=output_backend,
+            )
+            if i > 0:
+                figs.append(self.plot_h12_gwss_track(x_range=figs[0].x_range, **params))
+            else:
+                figs.append(self.plot_h12_gwss_track(**params))
+
+            figs[-1].xaxis.visible = False
+
+        # Plot genes.
+        fig2 = self.plot_genes(
+            region=contig,
+            sizing_mode=sizing_mode,
+            width=width,
+            height=genes_height,
+            x_range=figs[0].x_range,
+            show=False,
+            output_backend=output_backend,
+        )
+
+        figs.append(fig2)
+
+        # Combine plots into a single figure.
+        fig = bokeh.layouts.gridplot(
+            figs,
+            ncols=1,
+            toolbar_location="above",
+            merge_tools=True,
+            sizing_mode=sizing_mode,
```

Comment:
```suggestion
            sizing_mode=sizing_mode,
            toolbar_options=dict(active_inspect=None),
```

Apply solution from #633 to avoid warnings.